src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.spec.js

```diff
@@ -96,7 +98,7 @@ describe('FilePreview.vue', () => {
 		})
 
 		test('renders file preview for guests', async () => {
-			propsData.link = 'https://localhost/nc-webroot/s/xtokenx'
+			propsData.file.link = 'https://localhost/nc-webroot/s/xtokenx'
```

Comment:
check all `propsData.` changes in tests, some are missing adding a `file` like here